bundles/org.openhab.binding.guntamatic/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,596 @@
+			<channel id="controlExtraWwHeat2" typeId="controlExtraWwHeat">
+				<label>Trigger Extra Warm Wather Circle 2</label>
+			</channel>
+		</channels>
+		<properties>
+			<property name="boilerApproval">K0010</property>
+			<property name="program">PR001</property>
+			<property name="heatCircProgram">HKx01</property>
+			<property name="wwHeat">BKx06</property>
+			<property name="extraWwHeat">ZKx06</property>
+		</properties>
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the Guntamatic Heating System</description>
+			</parameter>
+			<parameter name="key" type="text" required="false">
+				<context>key</context>
+				<label>Key</label>
+				<description>Optional, but required to read protected parameters and to control the Guntamatic Heating System.
+					The
+					key needs to be reqested from Guntamatic support.</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="1" required="false">
+				<label>Refresh Interval</label>
+				<description>Interval the Guntamatic Heating System is polled in seconds. Default: 60s</description>
+			</parameter>
+			<parameter name="encoding" type="text" required="false">
+				<label>Encoding</label>
+				<description>Code page used by the Guntamatic Heating System. Default: 'windows-1252'</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+
+	<channel-type id="controlProgram" advanced="false">
+		<item-type>String</item-type>
+		<label>Set Program</label>
+		<description>Set Program of the Guntamatic Heating System (OFF, NORMAL, WARMWATER, MANUAL)</description>
+		<state>
+			<options>
+				<option value="0">OFF</option>
+				<option value="1">NORMAL</option>
+				<option value="2">WARMWATER</option>
+				<option value="8">MANUAL</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="controlBoilerApproval" advanced="true">
+		<item-type>String</item-type>
+		<label>Set Boiler Approval</label>
+		<description>Set Boiler Approval of the Guntamatic Heating System (AUTO, OFF, ON)</description>
+		<state>
+			<options>
+				<option value="0">AUTO</option>
+				<option value="1">OFF</option>
+				<option value="2">ON</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="controlHeatCircProgram" advanced="true">
+		<item-type>String</item-type>
+		<label>Set Program of Heat Circle 0...8</label>
+		<description>Set Program of Heat Circle 0...8 of the Guntamatic Heating System (OFF, NORMAL, HEAT, LOWER)</description>
+		<state>
+			<options>
+				<option value="0">OFF</option>
+				<option value="1">NORMAL</option>
+				<option value="2">HEAT</option>
+				<option value="3">LOWER</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="controlWwHeat" advanced="true">
+		<item-type>String</item-type>
+		<label>Trigger Warm Wather Circle 0...2</label>
```

Comment:
Short label please.
What is "wather" ? Do you mean "watter" ?

Reply:
> What is "wather" ? Do you mean "watter" ?

Upps, it is "water" 😳

Reply:
> Short label please.

`<label>Trigger Warm Water Circle</label>`